flit-client/src/components/DialogLine/index.tsx

```diff
@@ -20,18 +23,26 @@ function makeTimestamp(stamp: Date):string {
 }
 
 export default function render(props: Props) {
+    let classNames:string = props.own ? "name ownName": "name"
+
+    console.log(props.user.avatar)
+    let avatar:string = props.user.avatar && props.user.avatar !== undefined && props.user.avatar !== "" ? `${avatarBase}/${props.user.avatar}` : `${avatarBase}/face.svg` 
+
     return <div className='dialogLine'>
         <div className='face'>
-            <img src={face} width={32} height={32} alt=''/>
+            <img src={avatar} width={32} height={32} alt=''/>
+        </div>
+        <div className={classNames}>
+            {props.user.name}
         </div>
-        <div className='name'>
-            {props.name}
+        <div className='status'>
+            <LineStatus status={props.msgStatus} />
         </div>
```

Comment:
I like showing the status information! One thing we might want to consider is whether we want to promote this data into the class information of the message, so we can show it via styling, and let it recede in the default cases where the message has been successfully sent / received.